lib/sidekiq/instrument/worker.rb

```diff
@@ -21,18 +21,22 @@ def perform
       self.class::METRIC_NAMES.each do |method, stat|
         stat ||= method
 
-        StatsD.gauge "shared.sidekiq.stats.#{stat}", info.send(method)
+        Statter.statsd.gauge("shared.sidekiq.stats.#{stat}", info.send(method))
+        Statter.dogstatsd&.gauge("sidekiq.#{stat}", info.send(method))
       end
 
       working = Sidekiq::ProcessSet.new.select { |p| p[:busy] == 1 }.count
-      StatsD.gauge "shared.sidekiq.stats.working", working
+      Statter.statsd.gauge('shared.sidekiq.stats.working', working)
+      Statter.dogstatsd&.gauge('sidekiq.working_count', working)
 
       info.queues.each do |name, size|
-        StatsD.gauge "shared.sidekiq.#{name}.size", size
+        Statter.statsd.gauge("shared.sidekiq.#{name}.size", size)
+        Statter.dogstatsd&.gauge('sidekiq.queue_size', size, tags: ["queue:#{name}"])
       end
 
       Sidekiq::Queue.all.each do |queue|
-        StatsD.gauge "shared.sidekiq.#{queue.name}.latency", queue.latency
+        Statter.statsd.gauge("shared.sidekiq.#{queue.name}.latency", queue.latency)
+        Statter.dogstatsd&.gauge('sidekiq.queue_latency', queue.latency, tags: ["queue:#{queue.name}"])
```

Comment:
Maybe for the DogStatsD, they should be changed to `sidekiq.queue.<latency or size>`? Thoughts?

Reply:
I like it 👍